scipy/sparse/_data.py

```diff
@@ -370,15 +389,19 @@ def argmax(self, axis=None, out=None):
             This argument is in the signature *solely* for NumPy
             compatibility reasons. Do not pass in anything except for
             the default value, as this argument is not used.
+        explicit : {False, True} optional
+            Indicates whether the nonzero entries of the matrix should be
+            considered. If a row/column is empty, a zero will be returned
+            to indicate it contains no nonzero values.
 
         Returns
         -------
         ind : numpy.matrix or int
             Indices of maximum elements. If matrix, its size along `axis` is 1.
         """
-        return self._arg_min_or_max(axis, out, np.argmax, np.greater)
+        return self._arg_min_or_max(axis, out, np.argmax, np.greater, explicit)
 
-    def argmin(self, axis=None, out=None):
+    def argmin(self, axis=None, out=None, explicit=False):
```

Comment:
New arguments should be keywords only.

```suggestion
    def argmin(self, axis=None, out=None, *, explicit=False):
```